scrabble.rb

```diff
@@ -0,0 +1,45 @@
+  require_relative 'score/scrabble_score'
+  require "sinatra"
+  require "sinatra/reloader"
+
+  class ScrabbleSite < Sinatra::Base
+    register Sinatra::Reloader
+
+    get "/" do
+      erb :index
+    end
+
+    get "/score" do
+      erb :score
+    end
+
+    get "/score_words" do
+      erb :score_words
+    end
+
+    def output_score(word)
```

Comment:
This method is a bit out of place. I see that it's setting multiple instance variables so you have a few options:
1. It is setting a string with some variables, so it may actually make more sense to do this in your view (output)
2. You could just set this as a private method below your action methods to prevent confusion
3. Move this do the Scrabble class and have it return the score description